common/setups/rasr/util/rasr.py

```diff
@@ -198,5 +198,17 @@ def get_step_iter(self):
     def get_step_names_as_list(self):
         return list(self._step_names_args.keys())
 
+    def get_non_gmm_steps_as_list(self):
```

Comment:
why do we need this distinction into `gmm_steps` and `non_gmm_steps`? 

does it make sense to hardcode the list of non_gmm_steps to `["forced_align"]` are there no other non_gmm_steps?

Reply:
So our idea (maybe the naming is off) is that non_gmm steps are steps that do not (per default) include some gmm training where we have guaranteed alignments and mixture and thus these outputs cannot be used (since they dont exist)

Allowed steps right now are: "extract, mono, cart, tri, vtln, sat, vtln+sat, forced_align" 
I am not sure how to handle extract, because instantly calling output after extract should not be usual behavior. Maybe add this here aswell?

Reply:
ok, good idea.. But I'll let you guys at the chair decide on the design here